src/components/HeartbeatBar.vue

```diff
@@ -184,11 +184,11 @@ export default {
             }
 
             if (seconds < tolerance) {
-                return "now";
+                return (this.$t("now"));
             } else if (seconds < 60 * 60) {
-                return (seconds / 60).toFixed(0) + "m ago";
+                return (seconds / 60).toFixed(0) + "m";
             } else {
-                return (seconds / 60 / 60).toFixed(0) + "h ago";
+                return (seconds / 60 / 60).toFixed(0) + "h";
```

Comment:
Please add a translation instead of just removing it ^^

Reply:
Because I wanted to do it like the beginning of the bar and add the key "ago." However, I realized that by doing this, both the "now"  key was also displayed

![image](https://github.com/louislam/uptime-kuma/assets/70776486/d5fda26e-628e-469c-bd71-43bec91ac977)

Reply:
So I deleted it to leave only the 'now' or '1m' at the end without displaying the 'ago'

![image](https://github.com/louislam/uptime-kuma/assets/70776486/325ddf2a-8e9c-4336-a695-aa22be30f189)